packages/react-pagination/docs/react-pagination.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Pagination } from "../lib/react-pagination";
+
+export default {
+	title: "Components/Pagination",
+	component: Pagination,
+};
+
+let childrenArray = new Array();
+for (let i = 1; i <= 34 /* from here you can change the number of elements that can be inputed */; ++i)
+	childrenArray.push(<h5 key={i}>This is the sample data that I am working with. Id number is {i}</h5>);
+
+export const primary = args => (
+	<Pagination {...args}>
+		<div>Page</div>
+	</Pagination>
+);
```

Comment:
@Gowtham369 here I can find 2 ways of adding content to pagination. Please, split them into two examples as we do with the Button component, for example, that way back-end developers can have a better understanding of the difference. You can even split them in 3 ways: one through props, another as children content, and another that could be this one which is a mix of both.